config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Locations::Engine.routes.draw do
+  resources :floors
```

Comment:
As long as we avoid using "new" as a code it's not necessary, which shouldn't be a problem for floors, since we're building this table from scratch. I think it's better continue this pattern for consistency though.